projects/plugins/jetpack/extensions/blocks/paywall/edit.js

```diff
@@ -29,12 +36,27 @@ function PaywallEdit( { className } ) {
 	};
 
 	return (
-		<div className={ className }>
-			<span style={ style }>
-				{ text }
-				<Icon icon={ arrowDown } size={ 16 } />
-			</span>
-		</div>
+		<>
+			<div className={ className }>
+				<span style={ style }>
+					{ text }
+					<Icon icon={ arrowDown } size={ 16 } />
+				</span>
+			</div>
+			<InspectorControls>
+				<PanelBody
+					className="jetpack-subscribe-newsletters-panel"
+					title={ __( 'Newsletter visibility', 'jetpack' ) }
+					icon={ <JetpackLogo showText={ false } height={ 16 } logoColor="#1E1E1E" /> }
```

Comment:
Let's use the new shared component from https://github.com/Automattic/jetpack/pull/32257

Reply:
good call